src/context.tsx

```diff
@@ -90,30 +88,76 @@ const reducer = (state: State, action: Record<string, any>): State => {
         ...state,
         showCookieBanner: true
       };
-
-      case "OPEN_ANALYZE_POPUP":
-        return {
-          ...state,
-          showAnalyzePopup: true
-        };
-
+    case "OPEN_ANALYZE_POPUP":
+      return {
+        ...state,
+        showAnalyzePopup: true
+      };
     case "CLOSE_ANALYZE_POPUP":
       return {
         ...state,
         showAnalyzePopup: false
       };
-
     case "UPDATE_ESTIMATE_PREVALENCES":
       return {
         ...state,
         estimate_grade_prevalences: action.payload
       };
-    case "SELECT_EXPLORE_OR_ANALYZE":
+    case "SAVE_PAGE_STATE": {
+      const isExplorePageState = action.payload;
```

Comment:
note for later, we will eventually need this to support options for analyze, explore, and data tab. 

can use a switch statement instead of a boolean flag when that time comes (although maybe when that time comes we can put a bit more thought into the design as well)

Reply:
Also I'm confused, why do we need to get whether we're on explore or analyze from the action's payload, when we have a state variable called dataPageState that we're maintaining that should tell us that information already???

Reply:
excellent question